src/sxg_signer_list.c

```diff
@@ -17,6 +17,7 @@
 #include "libsxg/sxg_signer_list.h"
 
 #include <assert.h>
+#include <openssl/mem.h>
```

Comment:
Did you mean `openssl/crypto.h`?

Reply:
`mem.h` is only for [boringssl](https://github.com/google/boringssl) usually, we need nor `mem.h` and `crypto.h`.

Reply:
In general we should not rely on nested includes ([styleguide][]). That said, it's unclear to me how well the openssl's file hierarchy is organized, so I'm reluctant with the current code.

[styleguide]: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes